app/helpers/globalize_helper.rb

```diff
@@ -14,8 +14,12 @@ def display_translation?(locale)
     same_locale?(neutral_locale(I18n.locale), neutral_locale(locale)) ? "" : "display: none"
   end
 
-  def display_translation_for_resource?(resource, locale)
-    resource.translated_locales.include?(neutral_locale(locale)) || locale == I18n.locale
+  def display_translation_for_resource?(locale, resource = nil)
+    if resource.nil?
+      I18nContentTranslation.existing_languages.include?(neutral_locale(locale)) || locale == I18n.locale
+    else
+      resource.translated_locales.include?(neutral_locale(locale)) || locale == I18n.locale
+    end
```

Comment:
I think we usually avoid the `if something.nil? (...) else (...)` structure, and instead do something like:
```ruby
if resource
  resource.translated_locales.include?(neutral_locale(locale)) || locale == I18n.locale
else
  I18nContentTranslation.existing_languages.include?(neutral_locale(locale)) || locale == I18n.locale
end
```
Correct me if I'm wrong!

Reply:
I did thought about that but I think `nil?` usage makes it more explicit and therefore, easier to read